adapter_docs/overview.md

```diff
@@ -30,32 +30,33 @@ While these adapters have laid the foundation of the adapter-transformers librar
 .. important::
     In literature, different terms are used to refer to efficient fine-tuning methods.
     The term "adapter" is usually only applied to bottleneck adapter modules.
-    However, most efficient fine-tuning methods follow the same general idea of inserting a small set of new parameters and by this "adapting" the pre-trained LM to a new task.
+    However, most efficient fine-tuning methods follow the same general idea of inserting a small set of new parameters and, by this, "adapting" the pre-trained LM to a new task.
     In adapter-transformers, the term "adapter" thus may refer to any efficient fine-tuning method if not specified otherwise.
 ```
 
 In the remaining sections, we will present how adapter methods can be configured in `adapter-transformers`.
-The next two pages will then present the methodological details of all currently supported adapter methods.
+The following two pages will offer the methodological details of all currently supported adapter methods.
 
 ## Table of Adapter Methods
 
 The following table gives an overview of all adapter methods supported by `adapter-transformers`.
 Identifiers and configuration classes are explained in more detail in the [next section](#configuration).
+TODO: update links
```

Comment:
Still todo?

Reply:
Not this is resolved, forgot to remove the todo, thanks for spotting